google/resource_compute_image.go

```diff
@@ -195,3 +213,9 @@ func resourceComputeImageDelete(d *schema.ResourceData, meta interface{}) error
 	d.SetId("")
 	return nil
 }
+
+func resourceComputeImageImportState(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	d.Set("create_timeout", computeImageCreateTimeoutDefault)
```

Comment:
Agree for the ForceNew part, I will do it.

However, if we don't set `create_timeout` to the default value on import, then, after importing, the user will always have a "plan diff" with:
  ~ google_compute_image.web-image
      create_timeout: "" => "4"

The update is fast (only update the value locally). I am wondering if this is the expected behavior that after an update, we may have a diff that we can't fix without either running apply or fiddling with the state file.

Thoughts?